index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>First site</title>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/base.css" />
+    <link rel="stylesheet" href="./styles/extension.css" />
+  </head>
+  <body>
+    <!--section first with logo menu and first content-->
+    <header class="header container">
+      <div class="header__block block">
+        <h1 class="header__block--logo"><img src="./images/logo.svg" /></h1>
+        <nav class="nav">
+          <ul>
+            <li><a href="#team">The Team</a></li>
+            <li><a href="#pricing">Pricing</a></li>
+            <li><a href="#features">Features</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="main__container block">
+        <article class="main__container--article">
+          <h1 class="article__title bold">
+            " Beautiful Free Nova template "
+            <br />
+          </h1>
+          <h2 class="article__text light">
+            A top notch premium quality template for your next web project.
+            <br />
+          </h2>
+          <h3>
+            <button class="article__button">Download FREE!</button>
+          </h3>
+        </article>
+        <aside class="main__container--ads">
+          <img class="ads__img" src="images/screen.png" />
+        </aside>
+      </div>
+    </header>
+    <!--section second with specifications of product-->
+    <section id="features" class="specifications__container container">
+      <div class="specifications__top block">
+        <h1 class="specifications__header light">FEATURES</h1>
+      </div>
+      <div class="specifications__table block">
+        <section class="table">
+          <div class="specifications__table--background">
+            <img
+              src="/images/responsive.svg"
+              class="specifications__table--img"
```

Comment:
Jesli używamy modyfikatora z BEM to powinien on występować razme z Elementem czyli tutaj:
`specifications__table specifications__table--img`. Więcej znajdziesz tutaj: https://devmentor.pl/b/metodologia-bem-w-css-i-sass